x-pack/plugins/security_solution/public/common/components/charts/donutchart.tsx

```diff
@@ -71,6 +72,12 @@ export const DonutTextWrapper = styled(EuiFlexGroup)<
   max-width: 77px;
   position: absolute;
   z-index: 1;
+
+  &.risk-score {
```

Comment:
This CSS selector doesn't seem very maintainable. We shouldn't make references to specific pages or components from inside a generic donut chart. I would suggest to have a particular property for this behaviour, like `emptyDataTopPosition`, or reuse the class you receive as props in the selector:
```
&.${donutTextWrapperClassName} {
   ...
}
```

Reply:
Good point. I have updated the css rules here, so we can just pass a custom class and the styles  to the component.